### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # test
-Don't mind it
+Don't test it
```
Comment (Kmaschta) [inline README.md]: s/test/pouet/

Comment (Kmaschta) [inline README.md]: s/it/that/

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # test
-Don't mind it
+Don't test that
```
Comment (Kmaschta) [inline README.md]: s/test/pouet/

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 # test
-Don't mind it
+Don't pouet that
```
Comment (jpetitcolas) [inline README.md]: s/pouet/transluding scope is transcluded one/

Comment (jpetitcolas) [inline README.md]: s/pouet/transluding scope is transcluded one/

Comment (Kmaschta) [inline README.md]: It works!

Comment (Kmaschta) [inline README.md]: s/pouet/test/

File: README.md (hunk 2 above)
Comment (jpetitcolas) [inline README.md]: s/test/truc/

Comment (alexisjanvier) [inline README.md]: s/that/this

Comment (jpetitcolas) [inline README.md]: s/this/foo/

Comment (alexisjanvier) [inline README.md]: s/that/this/

File: README.md [hunk 4]
```diff
@@ -1,2 +1,2 @@
 # test
```
Comment (jpetitcolas) [inline README.md]: s/#/flying fish/

**Pull request closed without merging**